metron-platform/metron-elasticsearch/src/test/java/org/apache/metron/elasticsearch/integration/ElasticsearchIndexingIntegrationTest.java

```diff
@@ -102,11 +102,16 @@ public void setAdditionalProperties(Properties topologyProperties) {
     topologyProperties.setProperty("es.clustername", "metron");
     topologyProperties.setProperty("es.port", "9300");
     topologyProperties.setProperty("es.ip", "localhost");
-    topologyProperties.setProperty("indexing.writer.class.name", "org.apache.metron.elasticsearch.writer.ElasticsearchWriter");
+    topologyProperties.setProperty("indexing_writer_class_name", "org.apache.metron.elasticsearch.writer.ElasticsearchWriter");
   }
 
   @Override
   public String cleanField(String field) {
     return field;
   }
```

Comment:
I do agree it does seem like a violation of separation of concerns, but to be honest, I'm not terribly concerned about it in this case.

The copy-resources seems reasonable, if it's not too much work, but given this particular problem, I'm mostly content with just fixing it.